src/modules/transactions.test.js

```diff
@@ -0,0 +1,9 @@
+import { Transactions } from './transactions';
```

Comment:
would love to see a lot more tests in here

can/should we mock out the db?

Reply:
I've created additional tests. Realm can operate using memory for unit tests and the performance is good, i don't think it worth mocking it.